.circleci/config.yml

```diff
@@ -78,6 +78,27 @@ jobs:
       TOX_CONFIG: ./workflows
       DO_ENV: py27
 
+  test_amqp_postgres:
+    docker:
+      - image: circleci/python:2.7.14
+        environment:
+          TEST_DATABASE_URL: postgresql://cloudify:cloudify@localhost/cloudify_db
+      - image: rabbitmq:3.7.4
+      - image: circleci/postgres:9.5-alpine
+        environment:
+          POSTGRES_USER: cloudify
+          POSTGRES_PASSWORD: cloudify
+          POSTGRES_DB: cloudify_db
+    steps:
+      - checkout
+      - run: sudo apt-get install libldap-dev libsasl2-dev
```

Comment:
we need ldap here?

Reply:
Yeah =\ Unfortunately without it you can't install the rest-service venv